package.json

```diff
@@ -22,8 +22,7 @@
     "lodash": "^4.17.4",
     "moment": "^2.18.1",
     "moment-timezone": "^0.5.13",
-    "pegjs": "0.10.0",
```

Comment:
Do we really need it in dependencies or should it be moved to `devDependencies`? I guess it's used only for generating parsers in dev, am I wrong?